.github/workflows/cran-checks.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check CRAN status
+
+on:
+  schedule:
+    # Runs daily at 4:00 PM UTC (9:00 AM PST)
+    - cron: '0 16 * * *'
```

Comment:
Do you agree to launch the action in the evening? It's set to 6pm at the moment. I think we could set it earlier, e.g. 6:00 AM UTC so that you get notified at 8am (summer time) or 7am (winter time).